src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Analyzers/NewtonsoftReferenceAnalyzer.cs

```diff
@@ -52,8 +52,7 @@ public async Task AnalyzeAsync(IProject project, IDependencyAnalysisState state,
                 return;
             }
 
-            var references = await project.GetNuGetReferencesAsync(token).ConfigureAwait(false);
-            if (references.IsTransitivelyAvailable(NewtonsoftPackageName))
+            if (await project.NuGetReferences.IsTransitivelyAvailableAsync(NewtonsoftPackageName, token).ConfigureAwait(false))
```

Comment:
that looks a lot more readable than i was thinking when i left my earlier comment 😃